mw-webapp/src/utils/ThemeWorker.ts

```diff
@@ -106,6 +106,12 @@ const themedVariables: Record<string, Record<Theme, string>> = {
     // [Theme.DARK]: "rgb(152, 210, 235)",
     // [Theme.LIGHT]: "rgb(84,103,143)",
   },
+  hoverBackgroundColor: {
+    [Theme.DARK]: "#4E427F",
+    [Theme.LIGHT]: "#E6E3EC",
```

Comment:
we don't have such variable in the new design system (https://www.figma.com/file/ukyPUFcecdKzeBcdTHChYk/%D0%9F%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5?type=design&node-id=1924-38420&mode=design&t=alIpMQ162swaIMGM-0)

let's discuss it with the design -team

Reply:
its necessary because without this var chenched background open triger accordion when is open
![image](https://github.com/tritonJS826/masters-way/assets/112805668/b67d2700-07e9-4a6c-a799-a50571775755)
![image](https://github.com/tritonJS826/masters-way/assets/112805668/de7357b8-ff54-42b9-9d1d-3dcfd3100961)

Reply:
please discuss it with the designers and if it is necessary they should add it to the tokens

Reply:
have discuss, the designers agreed to add it to the tokens

Reply:
answered in personal message